schema/profile-to-establishment.js

```diff
@@ -0,0 +1,41 @@
+const BaseModel = require('./base-model');
+
+class ProfileToEstablishment extends BaseModel {
+  static get tableName() {
+    return 'profile_to_establishment';
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['establishmentId', 'profileId'],
+      additionalProperties: false,
+      properties: {
+        establishmentId: { type: 'integer' },
+        profileId: { type: 'string' },
+        createdAt: { type: 'string', format: 'date-time' },
+        updatedAt: { type: 'string', format: 'date-time' },
+        deleted: { type: ['string', 'null'] }
+      }
+    };
+  }
+
+  static get relationMappings() {
+    return {
+      profile: {
+        relation: this.BelongsToOneRelation,
```

Comment:
We don't need to define any relations here - only in the models we will be querying

Reply:
done